float32[] com_pos_x
float32[] com_pos_y
float32[] com_pos_z

float32[] left_arm_pos_x
float32[] left_arm_pos_y
float32[] left_arm_pos_z

float32[] right_arm_pos_x
float32[] right_arm_pos_y
float32[] right_arm_pos_z

float32[] fl_leg_pos_x
float32[] fl_leg_pos_y
float32[] fl_leg_pos_z

float32[] fr_leg_pos_x
float32[] fr_leg_pos_y
float32[] fr_leg_pos_z

float32[] hl_leg_pos_x
float32[] hl_leg_pos_y
float32[] hl_leg_pos_z

float32[] hr_leg_pos_x
float32[] hr_leg_pos_y
float32[] hr_leg_pos_z

int32[] swing_id
float32[] swing_t
float32[] time
